env.c

```diff
@@ -19,11 +19,19 @@ static struct treenode *pools;
 
 static void addpool(struct pool *);
 
```

Comment:
This blank line can be removed, just group all the forward declarations together.

Reply:
Fair enough.